.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+  push:
+    branches: [staging]
+  pull_request:
+    branches: [staging]
```

Comment:
after you restrict the deploy jobs to be on pushes only, you should add master to the list of `pull_request` branches to make sure the test job runs successfully

Reply:
Added `master` to the push and pull_requests lists.